build-monitor-acceptance/src/main/java/net/serenitybdd/integration/jenkins/process/JenkinsProcess.java

```diff
@@ -110,10 +110,10 @@ private ProcessBuilder process(Path executable, String... arguments) {
     private Process start(ProcessBuilder jenkinsProcessBuilder) throws IOException {
         Log.info("Starting Jenkins on port {}...", port);
 
-        Process process = jenkinsProcessBuilder.start();
-        process.getOutputStream().close();
+        Process jenkinsProcess = jenkinsProcessBuilder.start();
```

Comment:
Good point, because `process` shadows a member field of the same name. However, `jenkinsProcess` shadows another one.

No worries, I'll clean it up when merging.